pdf/creator/border.go

```diff
@@ -0,0 +1,372 @@
+package creator
```

Comment:
https://github.com/unidoc/unidoc/wiki/UniDoc-Developer-Guide#8-license-header-in-each-go-file